microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md

```diff
@@ -6,10 +6,23 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: devtools
-ms.date: 07/05/2023
+ms.date: 09/16/2023
 ---
 # Debug DOM memory leaks with the Detached Elements tool
 
+> [!IMPORTANT]
+> The **Detached Elements** tool is being deprecated and eventually removed.  The same functionality is now available in the **Memory** tool instead.  Deprecation period:
+>   * Microsoft Edge 130: The **Detached Elements** tool is still available, but the **Memory** tool is recommended instead.
+>   * Microsoft Edge 133: The **Detached Elements** tool is removed.
+>
+> The downstream-only CDP function `EdgeDOMMemory.getDetachedNodesIds` is deprecated; for the moment, it still works, but use `DOM.getDetachedDomNodes` instead.
```

Comment:
is this function deprecated in same sense as DE tool & same timeline?  @captainbrosset

Reply:
chrome devtools protocol (cdp)

Reply:
greatly clarified & spelled out specifics